src/App.js

```diff
@@ -1,29 +1,25 @@
-import logo from './logo.svg';
-import './App.css';
+import React from 'react';
+import Header from './header/header';
+import Calculator from './components/calculator';
 
-function App() {
-  return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit
-          {' '}
-          <code>src/App.js</code>
-          {' '}
-          and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
-  );
+class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+  }
+
+  render = () => (
+    <>
+      <Header />
+      <Calculator />
+    </>
+  )
 }
 
+// const App = () => (
+//   <div className="App">
+//     <Calculator />
+//   </div>
+// );
```

Comment:
- [ ] **[OPTIONAL]** I suggest you remove these commented codes to achieve a more cleaner codebase.